test/integration/command-logging-and-monitoring/command_logging_and_monitoring.prose.test.ts

```diff
@@ -0,0 +1,238 @@
+import { expect } from 'chai';
+
+import { DEFAULT_MAX_DOCUMENT_LENGTH, type Document } from '../../mongodb';
+
+describe('Command Logging and Monitoring Prose Tests', function () {
+  const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
+  const ELLIPSES_LENGTH = 3;
+  context('When no custom truncation limit is provided', function () {
+    /*
+        1. Configure logging with a minimum severity level of "debug" for the "command" component. 
+           Do not explicitly configure the max document length.
+
+        2. Construct an array docs containing the document {"x" : "y"} repeated 100 times.
+
+        3. Insert docs to a collection via insertMany.
+
+        4. Inspect the resulting "command started" log message and assert that 
+            the "command" value is a string of length 1000 + (length of trailing ellipsis).
+
+        5. Inspect the resulting "command succeeded" log message and assert that 
+           the "reply" value is a string of length <= 1000 + (length of trailing ellipsis).
+
+        6. Run find() on the collection where the document was inserted.
+
+        7. Inspect the resulting "command succeeded" log message and assert that 
+           the reply is a string of length 1000 + (length of trailing ellipsis).
+    */
+
+    it('should follow default truncation limit of 1000', {
+      metadata: {},
+      test: async function () {
+        const writable = {
+          buffer: [],
+          write(log) {
+            this.buffer.push(log);
+          }
+        };
+
+        // 1.
+        const client = this.configuration.newClient(
+          {},
+          {
+            [loggerFeatureFlag]: true,
+            mongodbLogPath: writable,
+            mongodbLogComponentSeverities: {
+              command: 'debug'
+            }
+          }
+        );
+
+        // 2.
+        const docs: Array<Document> = [];
+        for (let i = 0; i < 100; i++) {
+          docs.push({ x: 'y' });
+        }
+
+        // 3.
+        await client.db('admin').collection('test').insertMany(docs);
+
+        // remove sensitive commands for uniformity
+        const cleanedBuffer = [];
+        const afterInsertBufferLength = writable.buffer.length;
+        for (let i = 0; i < afterInsertBufferLength; i++) {
+          if (writable.buffer[i].command !== '{}' && writable.buffer[i].reply !== '{}') {
+            cleanedBuffer.push(writable.buffer[i]);
+          }
+        }
```

Comment:
The prose test does mention needing to do this, why do we have to?

Reply:
Paper trail: This was before the issue with the truthiness of the hello was resolved, it's been removed now.